SafeguardSessions/Src/SafeguardSessions.pq

```diff
@@ -1,24 +1,24 @@
-﻿[Version = "1.1.1"]
+﻿[Version = "1.2.0"]
```

Comment:
It is indeed a feature, but a breaking change due to change of input parameter types and order, so version should be 2.0.0.

Reply:
Done.